### DISCUSSION TITLE
Rich Text Editor for Textarea Component

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3233] [Q&A] [answered]
My team is working on a feature where users should be able to edit and format email content.

Does Paste have any guidelines/recommendations on using a rich text editors for the textarea component?  Any advice would be super helpful thank you!

**Comment by loreina** [comment, marked as answer]
Hi @tonge495,

The [Chat Composer](https://paste.twilio.design/components/chat-composer) component might work for your use case, it wraps Lexical and has rich text functionality.
